src/init.c

```diff
@@ -414,13 +448,26 @@ int read_cards(char *suggestion)
 				/* Get first count string */
```

Comment:
I increase AVAILABLE_DESIGN to allow splace for custom cards (20).

Even changing the card names leads to networks not loading. I'll retrain as soon as I am done patching the current set of comments.